livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamChannelCard.kt

```diff
@@ -0,0 +1,142 @@
+package io.getstream.livestream.compose.ui
+
+import android.content.Context
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.ViewStream
+import androidx.compose.material.ripple.rememberRipple
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import io.getstream.chat.android.compose.ui.theme.ChatTheme
+import io.getstream.livestream.compose.R
+import io.getstream.livestream.compose.models.LiveStreamChannelItem
+
+/**
+ * This is a view component for custom Channel cards to be shown on Home screen.
+ *
+ * @param liveStreamChannelItem - The single item [LiveStreamChannelItem] that represents a channel
+ * @param modifier - Modifier for styling the card.
+ * @param isDarkTheme - this is a Boolean to decide if the card needs to show dark variant or a light, by default its false
+ * @param onCardClick - click handler to allow the call site to provide functionality when the Card itself is clicked.
+ */
+@Composable
+fun LiveStreamChannelCard(
+    liveStreamChannelItem: LiveStreamChannelItem,
+    modifier: Modifier = Modifier,
+    isDarkTheme: Boolean = false,
```

Comment:
This should be delegated through the app theme

Reply:
I need ur insight on how to achieve this, im not sure in compose how to do so